skimage/draw/draw.py

```diff
@@ -41,35 +41,35 @@ def _ellipse_in_shape(shape, center, radii, rotation=0.):
     return np.nonzero(distances < 1)
 
 
-def ellipse(r, c, r_radius, c_radius, shape=None, rotation=0.):
-    """Generate coordinates of pixels within ellipse.
+def ellipse(r, c, r_radius, c_radius, orientation=0, shape=None, rotation=None):
+    """Generate coordinates of pixels corresponding to an ellipse.
 
     Parameters
     ----------
-    r, c : double
-        Centre coordinate of ellipse.
-    r_radius, c_radius : double
-        Minor and major semi-axes. ``(r/r_radius)**2 + (c/c_radius)**2 = 1``.
-    shape : tuple, optional
+    r, c : float
+        Center coordinate of the ellipse.
+    r_radius, c_radius : float
+        Minor and major semi-axes, where ``(r/r_radius)**2 + (c/c_radius)**2 = 1``.
+    orientation : float, optional (default : 0)
+        Major axis orientation in counter clockwise direction as radians.
```

Comment:
Semi-major axis orientation. imho since we are moving to unify everything in r/c coordinates, this should be counter-clockwise orientation from vertical... Is that doable?

Reply:
I'll study that. Is this the way we do in `EllipseModel()` / `ellipse_perimeter`?